tests/integration/test_ttl_move/test.py

```diff
@@ -1635,7 +1635,7 @@ def optimize_table(num):
         optimize_table(20)
 
         if positive:
-            assert check_used_disks_with_retry(node1, name, set(["external"]), 50)
+            assert check_used_disks_with_retry(node1, name, set(["external"]), 100)
```

Comment:
Does increasing number of retries fixes something? I doubt

Reply:
Anyway it still fails - https://s3.amazonaws.com/clickhouse-test-reports/66671/2f00c962711e13ca00af324366421fe4593b4ce6/integration_tests__tsan__[5_6].html

Reply:
I don't even understand why retries are needed after OPTIMIZE FINAL

Reply:
https://github.com/ClickHouse/ClickHouse/pull/67747